t/t-credentials.sh

```diff
@@ -290,6 +290,40 @@ begin_test "credentials skips wwwauth[] with option"
 )
 end_test
 
+begin_test "credentials can authenticate with Bearer auth"
+(
+  set -e
+  git credential capability </dev/null | grep "capability authtype" || exit 0
+
+  reponame="auth-bearer-token"
+  setup_remote_repo "$reponame"
+
+  printf "Bearer::token" > "$CREDSDIR/127.0.0.1--$reponame"
+
+  clone_repo "$reponame" "$reponame"
+  git checkout -b new-branch
+
+  git lfs track "*.dat" 2>&1 | tee track.log
+  grep "Tracking \"\*.dat\"" track.log
+
+  contents="b"
+  contents_oid=$(calc_oid "$contents")
+
+  printf "%s" "$contents" > b.dat
+  git add b.dat
+  git add .gitattributes
+  git commit -m "add b.dat"
+
+  GIT_TERMINAL_PROMPT=0 GIT_TRACE=1 GIT_TRANSFER_TRACE=1 GIT_CURL_VERBOSE=1 git push origin new-branch 2>&1 | tee push.log
+  grep "Uploading LFS objects: 100% (1/1), 1 B" push.log
+  echo "approvals:"
```

Comment:
Minor thought: these `echo` statements could possibly just be inline comments, as I don't think anything output to `stdout` is usually visible when running the tests.